sdk/resourcemanager/azure-resourcemanager-appplatform/src/main/java/com/azure/resourcemanager/appplatform/models/SpringService.java

```diff
@@ -91,11 +93,18 @@
      */
     Mono<TestKeys> enableTestEndpointAsync();
 
+    /** @return default Configuration Service for Enterprise Tier */
+    SpringConfigurationService getDefaultConfigurationService();
+
+    /** @return the entry point of Tanzu Configuration Service for Enterprise Tier */
+    SpringConfigurationServices configurationServices();
```

Comment:
Do we usually expose this `SpringConfigurationServices`? What can use do with it if they get the instance?

Reply:
Currently only `default` Configuration Service will be created for the user, so this one here is obsolete, will remove it.